scalafix-tests/output/src/main/scala/test/NoExtendsApp.scala

```diff
@@ -0,0 +1,51 @@
+package test
+
+object Main {
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+trait Something
+object Main2 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main3 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main4 extends Something
```

Comment:
Is this expected? In these cases, I'm not sure what we should do honestly. Removing App won't fix the problem however, what do you think about emitting a warning instead?

> Missing body for object that extends `scala.App`. See http://dotty.epfl.ch/docs/reference/dropped/delayed-init.html for possible workarounds.

It's possible to report warnings with `ctx.reporter.warn(pos, msg)`. There's currently no way to test reported messages.